adcc/ElectronicTransition.py

```diff
@@ -160,10 +160,27 @@ def transition_dipole_moment(self):
             warnings.warn("ADC(0) transition dipole moments are known to be "
                           "faulty in some cases.")
         dipole_integrals = self.operators.electric_dipole
-        return np.array([
+        ret = np.array([
             [product_trace(comp, tdm) for comp in dipole_integrals]
             for tdm in self.transition_dm
         ])
+        return ret
+
+    @cached_property
+    @mark_excitation_property()
+    @timed_member_call(timer="_property_timer")
+    def s2s_transition_dipole_moments(self):
```

Comment:
This should not be here, it's already implemented [here](https://github.com/adc-connect/adcc/blob/e58bc07cee2a47fb5967ca46e50a66c27ef9bda9/adcc/State2States.py#L76).
BTW transition dipole moments should not include the permanent ground state dipole moment 😄